### ISSUE TITLE
Custom title defined in _sidebar.md is overriden by its name

### ISSUE THREAD

**Issue by ustc-zzzz** [docsifyjs/docsify#1412]
#### Steps to reproduce

```
git clone https://github.com/ustc-zzzz/docsify-issue-1412-demo.git
cd docsify-issue-1412-demo
docsify serve
```

#### What is current behaviour

Page title is `Unwanted Hello World Title` now.

#### What is the expected behaviour

Page title should be `Expected Hello World Title`.

#### Other relevant information

Here is the content of `_sidebar.md`:

```markdown
* [Unwanted Hello World Title](/ "Expected Hello World Title")
```

The `title` attribute is declared while [it is overriden by its name](https://github.com/docsifyjs/docsify/pull/1286/files#diff-4eec5da5456a3851c3cec01f17ed0d912f176c887eb1e232b3430ffe7e90b24fR89).

- [x] Bug does still occur when all/other plugins are disabled?

- Your OS: `Linux ZZZ-PC-4 5.9.0-1-amd64 #1 SMP Debian 5.9.1-1 (2020-10-17) x86_64 GNU/Linux`
- Node.js version: `v12.14.0`
- npm/yarn version: `6.14.5`
- Browser version: `Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.111 Safari/537.36`
- Docsify version: `4.11.6`
- Docsify plugins: `None`

#### Please create a reproducible sandbox 

https://ustc-zzzz.github.io/docsify-issue-1412-demo/

#### Mention the docsify version in which this bug was not present (if any)

`4.11.4`

**Comment by sy-records** [comment]
Thanks for your feedback, fixed via #1360.

**Comment by tpluscode** [comment]
Has this been released?
